### PR TITLE
modules 24, 25, 26, 27, 28, 29, 30, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 53 and 54 fully shiftable

### PR DESCRIPTION
**Pull request by red031000**

### CODE REVIEW

File: arm9/modules/27/asm/module_27.s [hunk 1]
```diff
@@ -1449,13 +1449,31 @@ _02255338: .word 0x00000669
...
+MOD27_0225533C: ; 0x0225533C
+	.word MOD27_02254948, MOD27_02254988, MOD27_02254A14
+
+	.global MOD27_02255348
+MOD27_02255348: ; 0x02255348
+	.byte 0x40, 0x00, 0x24, 0x00
 	.byte 0xA0, 0x00, 0x24, 0x00, 0x40, 0x00, 0x54, 0x00, 0xA0, 0x00, 0x54, 0x00, 0x40, 0x00, 0x84, 0x00
-	.byte 0xA0, 0x00, 0x84, 0x00, 0x04, 0x00, 0x08, 0x00, 0x10, 0x00, 0x08, 0x00, 0x04, 0x00, 0x0E, 0x00
-	.byte 0x10, 0x00, 0x0E, 0x00, 0x04, 0x00, 0x14, 0x00, 0x10, 0x00, 0x14, 0x00, 0x00, 0x00, 0x00, 0x00
+	.byte 0xA0, 0x00, 0x84, 0x00
```
Comment (PikalaxALT) [inline arm9/modules/27/asm/module_27.s:1461]: `.short` here

File: arm9/modules/27/asm/module_27.s [hunk 2]
```diff
@@ -1449,13 +1449,31 @@ _02255338: .word 0x00000669
...
+
+	.global MOD27_02255360
+MOD27_02255360: ; 0x02255360
+	.byte 0x04, 0x00, 0x08, 0x00, 0x10, 0x00, 0x08, 0x00, 0x04, 0x00, 0x0E, 0x00
+	.byte 0x10, 0x00, 0x0E, 0x00, 0x04, 0x00, 0x14, 0x00, 0x10, 0x00, 0x14, 0x00
```
Comment (PikalaxALT) [inline arm9/modules/27/asm/module_27.s:1466]: `.short` here

File: arm9/modules/28/asm/module_28.s [hunk 3]
```diff
@@ -2965,27 +2965,87 @@ _02255E74: .word 0x00000163
...
+
+	.global MOD28_02255EA8
+MOD28_02255EA8: ; 0x02255EA8
+	.word MOD28_022553B4, MOD28_0225549C, MOD28_022555B4, MOD28_02255690
+	.word MOD28_02255768, MOD28_0225580C
+
+	.global MOD28_02255EC0
+MOD28_02255EC0: ; 0x02255EC0
+	.byte 0x30, 0x00, 0x2C, 0x00, 0xB0, 0x00, 0x2C, 0x00
 	.byte 0x30, 0x00, 0x5C, 0x00, 0xB0, 0x00, 0x5C, 0x00, 0x30, 0x00, 0x8C, 0x00, 0xB0, 0x00, 0x8C, 0x00
```
Comment (PikalaxALT) [inline arm9/modules/28/asm/module_28.s:2998]: `.short` here

File: arm9/modules/28/asm/module_28.s [hunk 4]
```diff
@@ -2965,27 +2965,87 @@ _02255E74: .word 0x00000163
...
+	.balign 4
+	.global MOD28_02255F94
+MOD28_02255F94: ; 0x02255F94
+	.asciz " icon[%d] TchHate\n"
+
+	.balign 4
+	.global MOD28_02255FA8
+MOD28_02255FA8: ; 0x02255FA8
+	.asciz " icon[%d] Reset!!\n"
+	.byte 0x00, 0x00, 0x00, 0x00, 0x00
```
Comment (PikalaxALT) [inline arm9/modules/28/asm/module_28.s:3051]: Trailing `.byte 0`s can be omitted as they are filled in by alignment (to 0x20).

File: arm9/modules/34/asm/module_34.s [hunk 5]
```diff
@@ -585,17 +585,34 @@ _02254C54:
...
+	.global MOD34_02254C7C
+MOD34_02254C7C: ; 0x02254C7C
+	.byte 0x10, 0x40, 0x10, 0x40
 	.byte 0x10, 0x40, 0xA0, 0xD0, 0x48, 0x78, 0x10, 0x40, 0x48, 0x78, 0xA0, 0xD0, 0x80, 0xB0, 0x10, 0x40
-	.byte 0x80, 0xB0, 0xA0, 0xD0, 0x42, 0x00, 0x54, 0x00, 0x22, 0x01, 0x34, 0x01, 0x02, 0x02, 0x14, 0x02
+	.byte 0x80, 0xB0, 0xA0, 0xD0
+
+	.global MOD34_02254C94
+MOD34_02254C94: ; 0x02254C94
+	.byte 0x42, 0x00, 0x54, 0x00, 0x22, 0x01, 0x34, 0x01, 0x02, 0x02, 0x14, 0x02
```
Comment (PikalaxALT) [inline arm9/modules/34/asm/module_34.s:605]: `.short` here

File: arm9/modules/39/asm/module_39.s [hunk 6]
```diff
@@ -1445,30 +1445,63 @@ _022552FA:
...
+	.byte 0x80, 0xA8, 0x5C, 0x84, 0x82, 0xA4, 0x18, 0x48, 0x82, 0xA4, 0x98, 0xC8
+
+	.global MOD39_0225530C
+MOD39_0225530C: ; 0x0225530C
+	.word MOD39_022549F0, MOD39_02254A30, MOD39_02254BE8
+
+	.global MOD39_02255318
+MOD39_02255318: ; 0x02255318
+	.byte 0xFE, 0xFF, 0xFF, 0xFF, 0x10, 0x00, 0x00, 0x00
 	.byte 0x10, 0x00, 0x00, 0x00, 0xFB, 0xFF, 0xFF, 0xFF, 0x6B, 0x06, 0x00, 0x00, 0xFF, 0xFF, 0xFF, 0xFF
```
Comment (PikalaxALT) [inline arm9/modules/39/asm/module_39.s:1460]: These appear to be `.word`s

**Comment by red031000** [comment]
that's all the poketch modules, finally
ready for review

File: arm9/modules/43/asm/module_43.s [hunk 7]
```diff
@@ -1128,8 +1128,14 @@ _022550A2:
...
 	.section .rodata
-	; 0x022550A8
-	.byte 0x8D, 0x49, 0x25, 0x02, 0xD9, 0x49, 0x25, 0x02, 0x51, 0x4A, 0x25, 0x02, 0x05, 0x14, 0x05, 0x14
+
+	.global MOD43_022550A8
+MOD43_022550A8: ; 0x022550A8
+	.word MOD43_0225498C, MOD43_022549D8, MOD43_02254A50
+
+	.global MOD43_022550B4
+MOD43_022550B4: ; 0x022550B4
```
Comment (PikalaxALT) [inline arm9/modules/43/asm/module_43.s:1137]: Arrange in rows of 2

File: arm9/modules/45/asm/module_45.s [hunk 8]
```diff
@@ -1064,20 +1070,47 @@ _02255008: .word 0x02255118
...
+	.byte 0x48
+	.byte 0x1F, 0x1C, 0x1F, 0x1E, 0x1F, 0x1E, 0x1F, 0x1F, 0x1E, 0x1F, 0x1E, 0x1F
+
+	.global MOD45_022550B8
+MOD45_022550B8: ; 0x022550B8
+	.byte 0x00, 0x00, 0x00, 0x00
+	.byte 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x00, 0x00
+
+	.global MOD45_022550C6
+MOD45_022550C6: ; 0x022550C6
```
Comment (PikalaxALT) [inline arm9/modules/45/asm/module_45.s:1086]: `.short` here

File: arm9/modules/47/asm/module_47.s [hunk 9]
```diff
@@ -1481,26 +1481,61 @@ _0225532C:
...
+MOD47_022553B0: ; 0x022553B0
+	.byte 0x00, 0x06, 0x08, 0x0B, 0x0C, 0x0E, 0x00, 0x00
+
+	.global MOD47_022553B8
+MOD47_022553B8: ; 0x022553B8
+	.word MOD47_02254938, MOD47_02254978, MOD47_022549E4, MOD47_02254AD4
+	.word MOD47_02254B38, MOD47_02254B88
+
+	.global MOD47_022553D0
+MOD47_022553D0: ; 0x022553D0
```
Comment (PikalaxALT) [inline arm9/modules/47/asm/module_47.s:1511]: Arrange in rows of 2

File: arm9/modules/47/asm/module_47.s [hunk 10]
```diff
@@ -1481,26 +1481,61 @@ _0225532C:
...
+MOD47_022553A4: ; 0x022553A4
+	.byte 0x09, 0x0D, 0x07, 0x12
+
+	.global MOD47_022553A8
+MOD47_022553A8: ; 0x022553A8
+	.byte 0x01, 0x02, 0x03, 0x04, 0x05, 0x00, 0x00, 0x00
+
+	.global MOD47_022553B0
+MOD47_022553B0: ; 0x022553B0
+	.byte 0x00, 0x06, 0x08, 0x0B, 0x0C, 0x0E, 0x00, 0x00
```
Comment (PikalaxALT) [inline arm9/modules/47/asm/module_47.s:1503]: `.balign 4`; strip trailing 0s

File: arm9/modules/48/asm/module_48.s [hunk 11]
```diff
@@ -953,14 +953,31 @@ _02254F5C: .word 0x000003E7
 	thumb_func_end MOD48_02254EF4
 
 	.section .rodata
-	; 0x02254F60
-	.byte 0x3E, 0x62, 0x58, 0x88, 0x4E, 0x72, 0x98, 0xC8, 0x56, 0x7A, 0x18, 0x48, 0xB1, 0x49, 0x25, 0x02
-	.byte 0xF1, 0x49, 0x25, 0x02, 0x6D, 0x4A, 0x25, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+
+	.global MOD48_02254F60
+MOD48_02254F60: ; 0x02254F60
```
Comment (PikalaxALT) [inline arm9/modules/48/asm/module_48.s:958]: Arrays passed to the first arg of MOD20_02254130 appear to be Nx4 u8 arrays. Arrange accordingly.

File: arm9/modules/53/asm/module_53.s [hunk 12]
```diff
@@ -4151,17 +4151,17 @@ MOD53_021D98A0: ; 0x021D98A0
...
-	.byte 0x01, 0x00, 0x00, 0x00, 0xE1, 0x86, 0x1D, 0x02, 0x01, 0x00, 0x00, 0x00, 0x05, 0x87, 0x1D, 0x02
-	.byte 0x01, 0x00, 0x00, 0x00, 0x9D, 0x87, 0x1D, 0x02, 0x01, 0x00, 0x00, 0x00, 0xD9, 0x87, 0x1D, 0x02
-	.byte 0x01, 0x00, 0x00, 0x00, 0xF9, 0x87, 0x1D, 0x02, 0x01, 0x00, 0x00, 0x00, 0x25, 0x88, 0x1D, 0x02
-	.byte 0x01, 0x00, 0x00, 0x00, 0x61, 0x88, 0x1D, 0x02, 0x01, 0x00, 0x00, 0x00, 0xB1, 0x88, 0x1D, 0x02
-	.byte 0x01, 0x00, 0x00, 0x00, 0x6D, 0x89, 0x1D, 0x02, 0x01, 0x00, 0x00, 0x00, 0xA1, 0x89, 0x1D, 0x02
-	.byte 0x01, 0x00, 0x00, 0x00, 0xC9, 0x89, 0x1D, 0x02, 0x01, 0x00, 0x00, 0x00, 0x09, 0x8A, 0x1D, 0x02
-	.byte 0x01, 0x00, 0x00, 0x00, 0x0D, 0x8A, 0x1D, 0x02, 0x01, 0x00, 0x00, 0x00, 0x2D, 0x8A, 0x1D, 0x02
-	.byte 0x01, 0x00, 0x00, 0x00, 0x51, 0x8A, 0x1D, 0x02, 0x01, 0x00, 0x00, 0x00, 0xB1, 0x8A, 0x1D, 0x02
-	.byte 0x01, 0x00, 0x00, 0x00, 0xF9, 0x8A, 0x1D, 0x02, 0x01, 0x00, 0x00, 0x00
+	.word 0x01, MOD53_021D8494, 0x00, MOD53_021D84EC
```
Comment (PikalaxALT) [inline arm9/modules/53/asm/module_53.s:4154]: Is this an Nx2 array?

**Pull request merged**